Activity_Selection/Activity_Selection.go

```diff
@@ -1,35 +1,49 @@
+/*
+The problem statement for Activity Selection is that "Given a set of n activities with their start and finish times, we need to select maximum number of non-conflicting activities that can be performed by a single person, given that the person can handle only one activity
+at a time." The Activity Selection problem follows a Greedy approach.
+*/
+
 package main
 
 import (
-	"fmt"
-	"strconv"
+    "fmt"
 )
 
-// this algorithm is from wikipage(https://en.wikipedia.org/wiki/Activity_selection_problem)
+// Function to compute the activities to be chosen
 func activitySelection(startTime []int, finishTime []int) {
-	// second step is to print first activity from sorted activity
-	fmt.Println("activity 1")
-	// do following in sorted activities:
-	// If the start time of this activity is greater than the finish time of previously
-	// selected activity then select this activity and print it
-	preSelected := 0
-	for i := 1; i < len(finishTime); i++ {
-		if startTime[i] > finishTime[preSelected] {
-			fmt.Println("activity", strconv.Itoa(i + 1))
-			preSelected = i
-		}
-	}
+    // second step is to print first activity from sorted activity
+    i := 0
+    fmt.Println("Following activities are selected: ")
+    fmt.Printf("%d ", i)
+    // do following in sorted activities:
+    // If the start time of this activity is greater than the finish time of previously
+    // selected activity then select this activity and print it
+    preSelected := 0
```

Comment:
preSelected and i are serving the same purpose. Using both of them can be confusing.

Reply:
yes, correct. 
I have changed it.

Reply:
Great. Logic and implementation looks good to me.

Reply:
Some minor fixes are needed in some files though. For eg. 4 space indentation, new line at the end of file. Blank space between logical blocks. 

Just fix them and it will be ready to be merged.

Reply:
Thanks. 
I have fixed. Please review